uint32 timeout_ms
uint32 region_x
uint32 region_y
uint32 width
uint32 height
---
sensor_msgs/Image image
sensor_msgs/CamInfo cam_info
